table/init.go

```diff
@@ -74,3 +78,15 @@ func Configure() {
 func SetCsCapacity(capacity int) {
 	csCapacity = capacity
 }
+
+func init() {
```

Comment:
@zjkmxy I want to allow the user to configure the FIB table implementation before YaNFD starts up, but it seems like `core.GetConfigStringDefault()` does not work for unit tests. Do you have any suggestions for how to work around this so that it works for both regular use and unit tests?

Or do we not want to expose this option, and if we want to try a different algorithm we just change the value in a file and recompile YaNFD?

Reply:
For this specific problem, I suggest you remove `init()` function, and let the main program manually creates this table. `init()` function is designed for initialize default configuration and check state. I don't think it is suitable for creating any actual objects that have complex dependency.

FYI: Not related to this refactoring, but there is a pattern called "dependency injection", which roughly means an object (like FIB) should accept arguments of components it depend on (like core.Config), instead of trying to access them directly. By doing so, test units can inject mock configuration/database/etc.